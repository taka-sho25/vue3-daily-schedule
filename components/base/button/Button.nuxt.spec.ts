import { describe, test, expect } from 'vitest';
import { renderSuspended } from '@nuxt/test-utils/runtime';

import Button from './index.vue';

describe('Button test', () => {
  test('default snapshot', async () => {
    const render = await renderSuspended(Button, {
      props: {
        type: 'button',
      },
      slots: {
        default: 'slot default',
      },
    });
    expect(render.html()).toMatchSnapshot();
  });
});
