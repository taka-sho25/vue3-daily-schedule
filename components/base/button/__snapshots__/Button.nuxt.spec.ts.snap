// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Button test > default snapshot 1`] = `"<div id="test-wrapper"><button type="button" class="button">slot default</button></div>"`;
